app/code/Magento/Catalog/view/adminhtml/web/catalog/product/attribute/unique-validate.js

```diff
@@ -10,11 +10,17 @@ define([
     'use strict';
 
     return function (config) {
```

Comment:
@omiroshnichenko I tried this but it's to complex for me at this moment. I don't even get how I need to make the file structure. Do I need to follow the exact path as in the module directory? Other modules use custom paths and are ignore the path of the module.